components/DaysOfTheWeek/DaysOfTheWeekContainer.jsx

```diff
@@ -2,55 +2,63 @@ import { useState } from "react";
 import { Center, Flex, GridItem, SimpleGrid, Text } from "@chakra-ui/react";
 import DaysOfWeekButton from "./DayOfTheWeekButton";
 import LinkButton from "../LinkButton/LinkButton";
+import useForm from "../../components/FormProvider";
 
 export default function DaysOfTheWeekContainer({
   setNumberOfDays,
   saveDataAndLogs,
   customHref,
   disabledDays,
 }) {
+  const { answers, _ } = useForm();
+  const [wfhDays, __] = useState(answers.wfhDays);
+  const [onsiteDays, ___] = useState(answers.onsiteDays);
+
   // initial state for days of the week has info if it's selected or not (instead of having 2 separate states)
   const [daysOfTheWeek, setDaysOfTheWeek] = useState([
     {
       id: 0,
       day: "Monday",
-      isSelected: false,
+      isSelected: wfhDays.includes("Monday") || onsiteDays.includes("Monday"),
```

Comment:
Hi @hqtan , look great, just wonder if we shoould refer to **day** key within the object instead of hard coding it?